pom.xml

```diff
@@ -1,99 +1,114 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-
     Sonatype Nexus (TM) Open Source Version
     Copyright (c) 2018-present Sonatype, Inc.
     All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
-
     This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
     which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
-
     Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
     of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
     Eclipse Foundation. All other trademarks are the property of their respective owners.
-
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.sonatype.nexus.plugins</groupId>
         <artifactId>nexus-plugins</artifactId>
-        <version>3.13.0-01</version>
+        <version>3.19.0-SNAPSHOT</version>
```

Comment:
@bhamail should this be `3.19.0-01` now that it's released?

Reply:
Yes, in fact we can now use `3.19.1-01`.

Reply:
in: 5f4baf6